openhands/resolver/README.md

```diff
@@ -0,0 +1,182 @@
+# OpenHands Github Issue Resolver 🙌
```

Comment:
It would also be helpful to include a `DEVELOPMENT.md` file. It would mostly be the same instructions as running `openhands-resolver` locally, except we would do the following

1. For installation run `poetry install`
2. For testings run the files directly  `poetry run python openhands/resolver/resolver_issue.py .......`

Reply:
I think this is a good idea, but maybe we can leave it for later?

Reply:
Sounds good!